controllers/storagecluster/storageclasses_test.go

```diff
@@ -7,7 +7,11 @@ import (
 	configv1 "github.com/openshift/api/config/v1"
 	api "github.com/red-hat-storage/ocs-operator/v4/api/v1"
 	"github.com/stretchr/testify/assert"
+
```

Comment:
```suggestion
```

Reply:
This is intended to create a split between groups, should I still remove?

Reply:
yes pls

Reply:
Done